src/JabbR-Core/Localization/ClientResourceManager.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace JabbR_Core.Localization
+{
+    public class ClientResourceManager
+    {
+        // Placeholder class. Will add logic for providing client resources to settings soon.
```

Comment:
This is a mix of #2 and #3 from the email, since this class exists now, go ahead and bring _clientResources into it and create a BuildClientResources there so everything is encapsulated in this class.